modules/ROOT/pages/salesforce-to-twilio.adoc

```diff
@@ -0,0 +1,454 @@
+
+. Navigate to the xref::https://workbench.developerforce.com/query.php[SOQL query builder]. (You have to log in with your Salesforce credentials.)
+. Select *Opportunity* as the Object.
+. Select *Amount*, *CloseDate*, *Id*, and *Name* from the *Fields* menu.
+. In the *Filter results by* fields, add `StageName = ‘Closed Won’`.
+. Test the query by using the *Query* button. You should get a list of all closed Opportunities. Your query should look like this : `SELECT Amount,CloseDate,Id,Name FROM Opportunity WHERE StageName = 'Closed Won'`
++
+image::soql-query-screen-1.png[]
+
+. Copy the query to your clipboard.
+. Navigate to the Streaming Push topics page from the *query* menu at the top of the workbench.
+. Create a new Push Topic by selecting the option *Create New*.
+. Name the Push Topic `ClosedOpps` and paste in the query that you built.
+. Save the Push Topic.
++
+image::streaming-push-topics.png[]
+
+Now, every time the query criterion is met, Salesforce will push an event into the `ClosedOpps` Push Topic.
+
+== Create a project in Flow Designer
+
+When you are working in Flow Designer, you create your Mule applications in projects. A project can contain only one Mule application.
+
+A Mule application consists of one or more _flows_. According to the MuleSoft glossary, a flow is "a simple, flexible mechanism that enables orchestration of services using the sophisticated message flow capabilities of Mule runtime engine. Also referred to as service orchestration." In Flow Designer, what this means is a sequence of _cards_ that represent core components in Mule Runtime or the operations of connectors. (Cards can represent a couple of other things, too; however, for this tutorial, you need know only about those two.)
+
+Data moves from the beginning of a flow to the end of the flow. Along the way, the cards can update systems, query systems, send messages, delete database rows, or do any of the things that core components and connectors can do.
+
+There is a good number of core components and there are many connectors. For this tutorial, you'll use these:
+
+* Salesforce connector
+* Set Variable component
+* Choice router component
+* Database connector
+* Set Payload component
+* Transform component
+* Twilio connector
+
+For this project, you'll create only a single flow.
+
+Follow these steps to create the project.
+
+. Log into Anypoint Platform.
+. Open Design Center and create a new Mule application.
+.. Click *Create* next to the search field.
+.. In the *Create a Mule Application* section of the overlay, click *Create New Application*.
+.. Name your project `Order-to-cash automation`.
+. In the *Let’s Get Started* wizard, click the link `Go straight to canvas`, which is in the bottom-left corner.
++
+*Result:* Flow Designer appears, an empty trigger card displayed in the middle of the canvas.
```

Comment:
just to further my comment from above @rddheath @jimiljojo it results in a poor 'FTUX' or empty canvas experience. Maybe this is something we can work on later though

Reply:
I think wizards that "dumb down" the first experience can interfere with the learning that users need to go through. I think users need to engage with the product as it is when they first use it, and be encouraged by good, clever guidance to engage with it.

Reply:
ok